spec/index.html

```diff
@@ -1210,6 +1246,15 @@ <h2>RDFS Interpretations</h2>
       x is in ICEXT(I(<code>rdfs:Datatype</code>)) then <span >&lt; x,
       I(<code>rdfs:Literal</code>) &gt; is in IEXT(I(<code>rdfs:subClassOf</code>))</span></p></td>
     </tr>
+    <tr>
+    <td class="semantictable" id="rdfssemcond11"><p>If
+      exist x,y,z such that RE(x,z,y)=r, or 
```

Comment:
I'm pretty sure this is a typo...
```suggestion
      exists x,y,z such that RE(x,z,y)=r, or 
```

Reply:
I guess it should remain as a plural verb.